cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -131,18 +136,30 @@ class LeadImpl extends ServerImpl with Lead with Logging {
 
       if (bootProperties.getProperty(Constant.ENABLE_ZEPPELIN_INTERPRETER,
         "false").equalsIgnoreCase("true")) {
-        /**
-         * This will initialize the zeppelin repl interpreter.
-         * This should be done before spark context is created as zeppelin
-         * interpreter will set some properties for classloader for repl
-         * which needs to be specified while creating sparkcontext in lead
-         */
-        val props: Properties = ZeppelinIntpUtil.initializeZeppelinReplAndGetConfig()
-        props.asScala.foreach(kv => conf.set(kv._1, kv._2))
-      }
-      logInfo("About to initialize SparkContext with SparkConf=" + conf.toDebugString)
 
+        try {
+
+          var zeppelinIntpUtilClass = Class.forName("org.apache.zeppelin.interpreter.ZeppelinIntpUtil")
```

Comment:
As ZeppelinIntpUtil class is used to call some static method but scala does not support to reflect static method I have used java reflections ( https://issues.scala-lang.org/browse/SI-6459 )